README.md

```diff
@@ -17,16 +17,16 @@
 
 `EuroCropsML` is a pre-processed and ready-to-use machine learning dataset for crop type classification of agricultural parcels in Europe.
 It consists of a total of **706,683** multi-class labeled data points with a total of **176** distinct classes.
-Each data point contains an annual time series of per parcel median pixel values of Sentinel-2 L1C reflectance data for the year 2021.
+Each data point contains an annual time series of per parcel median pixel values of Sentinel-1 data or Sentinel-2 L1C (top-of-atmosphere) reflectance data for the year 2021.  For Sentinel-1, we utilize the C-band Synthetic Aperture Radar (SAR) Ground Range Detected (GRD) data. Imagery is selected based on the orbit type available for the location, either ascending or descending. In terms of polarization, we use Interferometric Wide (IW) mode with VV (vertical polarization emission and reception) and VH (vertical polarization emission and horizontal reception) bands.
 The dataset is based on [Version 9](https://zenodo.org/records/10118572) of [`EuroCrops`](https://github.com/maja601/EuroCrops), an open-source collection of remote sensing reference data.
 
 For `EuroCropsML`, we acquired and aggregated data for the following countries:
 
-| Country      | Total number of datapoints  | Number of distinct classes | 
-|--------------|--------------------| -------------------------- |
-| Estonia      |  175,906 | 127 |
-| Latvia       | 431,143 | 103 |
-| Portugal     | 99,634 | 79 |
+| Country      | Total number of datapoints for Sentinel-2 | Total number of datapoints for Sentinel-1 | Number of distinct classes | 
+|--------------|-------------------------------------------|-------------------------------------------|----------------------------|
+| Estonia      | 175,906                                   | 154,742                                    | 127                        |
```

Comment:
Note: The number of Sentinel-1 still needs to be updated, once the data has been processed again. This one is missing the MultiPolygons.